sections/01_abstract.md

```diff
@@ -3,3 +3,14 @@
 Offspring of the [Deep Review](https://github.com/greenelab/deep-review).
 We cite the Deep Review like [@doi:10.1101/142760] or [@tag:deep_review].
 The source repository is [@url:https://github.com/greenelab/manubot-rootstock].
+
+Equations can be integrated and labeled like this `{{ '{{ \'{#eq:label}\' }}' }}` and then
+referenced using `{@eq:label}` like this for Equation @eq:line.
+
+$$ y = mx + b $$ {{ '{#eq:line}' }}
+
+Images can be included and labeled like `{{ '{{ \'{#fig:label}\' }}' }}` and then
```

Comment:
> wasn't sure if (or how) we'd like to implement them

I think definitely yes. I'm assuming you'd specify the jinja2 [environment here](https://github.com/greenelab/manubot-rootstock/blob/439bf03bd49e0ccf39c7e1689a71daa445c21400/build/references.py#L171-L173). Let me know if you get stuck and I can look more into it.